test/Array/pop4.js

```diff
@@ -0,0 +1,32 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (c) 2022 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const array = [1, 2, 3];
+
+const testRuns = 100_000;

```

Comment:
We shouldn't need this many test runs, 10 should be enough to trigger the bug when running with -mic:1 (which is used by the test runner)